app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Realm/RealmStory.java

```diff
@@ -12,95 +12,133 @@
  * @see RealmObject
  */
 public class RealmStory extends RealmObject implements Story{
-    private RealmPublication publication;
-    private String publicationDate;
-    private String lastUpdated;
-    private RealmList<RealmAuthor> author;
-    private RealmReactions reactions;
-    private String fullText;
-    private String blurb;
+    private String datePublished;
```

Comment:
+1 to this too! Yep, let's stick to the conventional member variable naming scheme! Thanks for catching that!